Branch-SDK/Branch-SDK/Requests/BranchOpenRequest.m

```diff
@@ -173,4 +176,52 @@ - (NSString *)getActionName {
     return @"open";
 }
 
+
+#pragma - Open Response Lock Handling
+
+
+//	Instead of semaphores, the lock is handled by scheduled dispatch_queues.
```

Comment:
nitpick: is a tab needed between the `//` and the comment text? on a broader note, perhaps we could start thinking of having a linter for the sdk and decide on a style.